frontend/src/services/songs.ts

```diff
@@ -17,6 +17,30 @@ export async function getUserDetails() {
   }
 }
 
+export async function toggleUserChordsDisplay(isShowingChords) {
+  try {
+    await axios.patch<User>(`/api/users/`, {
+      is_showing_chords: isShowingChords,
+    });
+  } catch (error) {
+    console.error(
+      `Couldn't toggle user's chords display preferences: ${error}`,
+    );
+  }
+}
+
+export async function toggleUserColumnsDisplay(columnsToDisplay) {
+  try {
```

Comment:
Toggle is kind of a funny verb for this one since it takes a number. I get that we’re only allowing 1 or 2 columns, but that may change in the future (and we’re not passing a boolean right now).

Reply:
Changed toggle to set.